stepflow-rs/Cargo.toml

```diff
@@ -9,7 +9,6 @@ repository = "https://github.com/riptano/stepflow"
 [workspace]
 resolver = "2"
 members = ["crates/*"]
-default-members = ["crates/stepflow-main"]
```

Comment:
This "default-members" is what caused `cargo test` to only test `crates/stepflow-main`. I think we can safely remove it, and it will be better / avoid unwanted surprises.